Added Contact Backend and  message framework

akshatjai258 opened the pull request.

Killshot667 commented:
can u please explain it to me before I merge. there are many things I didn't understand.
Also, make more commits. Try not to add so many things as part of a single commit.

akshatjai258 commented:
Ok sorry for not committing in between will do that for next ti,e.

Here first i have added few HTML files then i added contact modal and created contact form then aftet that I added message framework of django so that i can give user message about successful commenting.....
